<launch>
  <param name="use_sim_time" value="false"/>
  <node name="DetectorNode" pkg="deep_pose_estimator" type="DetectorNode.py" output="screen"/>
 <?ignore  <node name="TrackerNode" pkg="deep_pose_estimator" type="TrackerNode.py" /> ?>
 <?ignore  <node pkg="rosbag" type="play" name="player" output="screen" args="-s 160 -r 0.6 -q --clock /root/share/tf/dataset/warsaw/2019-08-09*"/> ?>

</launch>